### ISSUE TITLE
Urban street upgrades 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#191]
sivarajan ; BuckarooBanzai ; wsor ; SX 

Add mod to add realistic lines on roads 
https://forum.minetest.net/viewtopic.php?t=2984

cries in too many registered nodes https://monitoring.minetest.land/d/QcXQ62fZk/registered-counts?orgId=1&refresh=5s

Latest activity - July 10, 2021 - on the streets mod 
https://github.com/minetest-streets/streets 

Cheapies one that includes infrastructure has some bug fixes as well
And is used on most servers.

Fix this by cutting down on pointless node defs, and using hardware coloring where needed

That mod registers all nodes in the stairsplus mod resulting in a huge node-count. 
Preferred is street-sign mod from VanessaE instead of more cement/tar/concrete variants.

Anyways, if you wanted to compromise, you could port the street line overlay nodes themselves, only work on flat nodes, but gives you most functionality
Would take like 10 node defs

What about the existing tar node with stripes overlay?

That node bound simplified SAO thing sounded very good solution for problems like this...
Don't remember what they called it but something between particle and entity, should take a look if it got any traction...

Basically just this node here: https://pandorabox-io.github.io/pandorabox-mods/#/items/building_blocks:Tar with overlays from the mod: https://github.com/minetest-streets/streets/blob/master/textures/streets_dashed_center_line.png for example

https://github.com/minetest/minetest/issues/6963 not done yet, still cooking

**Comment by Klaranth** [comment]
BuckarooBanzai
https://github.com/BuckarooBanzay/basic_streets

Staged on the test-server and PR here: https://github.com/pandorabox-io/pandorabox-mods/pull/1878

Features:
Tar blocks with markings (colorable, rotateable, sawable)
Player speed increases on tar nodes (factor 1...
This also allows players to move 1.5 times faster on tar-nodes

so tar + lava boots == fun 
+ speed beacon = ?

**Comment by Klaranth** [comment]
Ruggila ; BuckarooBanzai
Yeah, I wasn't sure whether to add it or not, if the sub-block (slabs, stairs, microblocks, etc) nodes would make you fast too i think that would just encourage building with them resulting in microblocks-streets and other "cheat" street combinations...

oh, okay. I build my slab-tar street around my base a long time ago. Now I read about the speed-tar, and just tried it...  so I thought it has been overlooked. I'm fine with it intentionally not speeding up slabs / slopes / stairs.

**Comment by OgelGames** [comment]
I don't think it's necessary to have an issue open here about this, the PR is enough.

**Comment by BuckarooBanzay** [comment]
mod merged, closing this :+1:

**Comment by Klaranth** [comment]
> mod merged, closing this 👍

Merged with what? Where? :)

**Comment by S-S-X** [comment]
> > mod merged, closing this 👍
> 
> Merged with what? Where? :)

This from earlier comments:

> Staged on the test-server and PR here: pandorabox-io/pandorabox-mods#1878
